src/commands/index.js

```diff
@@ -62,7 +62,8 @@ export default () => {
     ['component-exit', 'ComponentExit', 'select-parent'],
     ['component-delete', 'ComponentDelete'],
     ['component-style-clear', 'ComponentStyleClear'],
-    ['component-drag', 'ComponentDrag']
+    ['component-drag', 'ComponentDrag'],
+    ['disable-js', 'DisableJs', 'disable-js']
```

Comment:
The third option is used to support old commands, so you don't need it here :)

Reply:
I tried to remove 3rd option, but I receive this warning `'disable-js' command not found` on running this command via panel button.